x-pack/plugins/enterprise_search/server/routes/app_search/analytics.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { RouteDependencies } from '../../plugin';
+
+const querySchema = {
+  start: schema.maybe(schema.string({ minLength: 10, maxLength: 10 })), // Date string, format 'YYYY-MM-DD'
+  end: schema.maybe(schema.string({ minLength: 10, maxLength: 10 })), // Date string, format 'YYYY-MM-DD'
+  tag: schema.maybe(schema.string()),
+};
+const queriesSchema = {
+  ...querySchema,
+  size: schema.maybe(schema.number({ max: 20 })),
```

Comment:
I maybe went a little overboard with the schema validation here heh, I can get rid of the minLength/maxLength/max checks if people prefer it

Reply:
I think the only issue I can think of with over-validating here is that we have these same validations *already* in ent-search. Which means we have to keep these rules in sync, and update in two places as rules change.

Reply:
Ya super true, I think I'll just drop these

Reply:
4ab77fc